supervisor/host/info.py

```diff
@@ -66,6 +68,13 @@ async def get_dmesg(self) -> bytes:
 
         return stdout
 
+    def get_disk_free_space(self, path: Optional[Union[str, Path]] = None) -> float:
```

Comment:
Let's make an which use fix the path_supervisor
```
@property
free_space(self) -> float:
```

And put the `get_disk_free_space` into hardware as helper with path.